pydis_site/apps/content/resources/guides/python-guides/discordpy_help_command.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Custom Help Command
+description: "Overwrite discord.py's help command to implement custom functionality"
+---
+
+First,  a [basic walkthrough](https://gist.github.com/InterStella0/b78488fb28cadf279dfd3164b9f0cf96) by Stella#2000 on subclassing the HelpCommand will provide some foundational knowledge required before attempting a more customizable help command.
+
+---
+
+## Custom Subclass of Help Command
+If this does not fit your needs and you require a more customizable help command, you can subclass HelpCommand and add individual command details. Below is a basic demonstration:
+
+```python
+class MyHelpCommand(commands.HelpCommand):
+    async def command_callback(self, ctx, *,command=None):
```

Comment:
Should we really recommend people do it through this one method? There's many more specific to what the help command was called with.

Reply:
Updated